modules/sandboxed-containers-deleting-catalog-subscription.adoc

```diff
@@ -0,0 +1,22 @@
+//Module included in the following assemblies:
+//
+// *uninstalling-sandboxed-containers.adoc
+
+[id="sandboxed-containers-deleting-catalog-subscription_{context}"]
+= Deleting the {sandboxed-containers-first} catalog subscription
+
+You can delete the {sandboxed-containers-first} catalog subscription to remove namespace access to the {sandboxed-containers-operator}.
+
+.Prerequisites
+
+* You have {product-title} 1.0.0 installed.
+* You have {product-title} 4.8 installed on your cluster.
+* You have installed the OpenShift CLI (oc).
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Navigate to the *Operators* → *OperatorHub* page.
+. Search for `{sandboxed-containers-first}` and then select it.
```

Comment:
"select it" sounds kind of odd here to me. Maybe "and then select the Operator"?